gommand.go

```diff
@@ -6,14 +6,26 @@ import (
 	"log"
 	"os"
 	"os/exec"
-	"strings"
+	"syscall"
 
 	"golang.org/x/tools/imports"
 )
 
-func run(fileName string) (string, error) {
-	out, err := exec.Command("go", "run", fileName).CombinedOutput()
-	return string(out), err
+func run(fileName string) (int, error) {
+	cmd := exec.Command("go", "run", fileName)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	err := cmd.Run()
+	if cmd.ProcessState != nil {
+		if ps, ok := cmd.ProcessState.Sys().(syscall.WaitStatus); ok {
+			// Currently, go run doesn't exit with program exit code.
+			// This will be fixed in this CL.
+			// https://go-review.googlesource.com/#/c/18271/
```

Comment:
> This will be fixed in this CL.

This comment should probably be updated. That CL was reviewed with -2, meaning it will not be merged (with strong reasons).

I agree with Russ that it's not valid to change exit code of